### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by petrochenkov**
With help of https://github.com/lucasdemarchi/codespell

r? @steveklabnik

### CODE REVIEW

File: src/libstd/fs/mod.rs [hunk 1]
```diff
@@ -1511,7 +1511,7 @@ mod tests {
         assert_eq!(v, b"foobar\0\0\0\0".to_vec());
 
         // Truncate to a smaller length, don't seek, and then write something.
-        // Ensure that the intermediate zeroes are all filled in (we're seeked
+        // Ensure that the intermediate zeroes are all filled in (we're sought
```
Comment (Manishearth) [inline src/libstd/fs/mod.rs]: I don't think this is correct. Probably should be "We have `seek`ed past the end of the file" or something.

File: src/libstd/old_io/fs.rs [hunk 2]
```diff
@@ -1532,7 +1532,7 @@ mod test {
                    b"foobar\0\0\0\0".to_vec());
 
         // Truncate to a smaller length, don't seek, and then write something.
-        // Ensure that the intermediate zeroes are all filled in (we're seeked
+        // Ensure that the intermediate zeroes are all filled in (we're sought
```
Comment (Manishearth) [inline src/libstd/old_io/fs.rs]: Ditto

**Comment by Manishearth** [comment]
LGTM aside from the small issue with the seek comments.

**Comment by steveklabnik** [comment]
Thank you!

**Pull request merged**